src/controllers/searchVideos.controller.js

```diff
@@ -1,8 +1,9 @@
 const fetchVideosRanks = require("../utils/fetchVideosRanks");
 const Query = require("../models/Query");
+const checkUserInputSpelling = require("../utils/checkSpelling");
```

Comment:
util끼리 모아주면 좋을 것 같습니다!
```suggestion
const fetchVideosRanks = require("../utils/fetchVideosRanks");
const checkUserInputSpelling = require("../utils/checkSpelling");
const Query = require("../models/Query");

```

Reply:
네! 바로 수정하겠습니다! 👍